docs/ADR-014-Cardano-API-WASM-library-for-browser.md

```diff
@@ -0,0 +1,167 @@
+
+# ADR-014: JavaScript API for Cardano via WASM-compiled cardano-api
+
+**Status:** Proposed
+
+**Date:** 2025-05-22
+
+## Context
+
+Decentralized Applications (DApps) are primarily utilized through web browsers. And Cardano DApps in Cardano would benefit from the extensive features provided by `cardano-api`.
+
+There already exist several JavaScript/TypeScript libraries and APIs that support DApps (e.g., Lucid Evolution, Helios, MeshJS, cardano-serialization-lib, cardano-sdk-js, Ogmios, TyphonJS), but porting `cardano-api` can offer advantages over other approaches:
+* **Comprehensiveness:** `cardano-api` is especially comprehensive in its functionalities, since it is developed in sync with `cardano-node`.
```

Comment:
This list describes the advantages of the proposed cardano-api WASM. This list does not really belong in the context section.

Reply:
Addressed here: https://github.com/input-output-hk/cardano-node-wiki/pull/75/commits/8691367cfdfee2b5387249cda10827d54185edbe